src/fragments/lib/storage/flutter/copy.mdx

```diff
@@ -0,0 +1,25 @@
+You can copy an existing file to a different key. This requires the user who initiates a copy operation to have read permission on the copy resource file.
```

Comment:
can we be more specific here and say that you can copy to a different location? Key is a super specific S3 terminology.